lib/autorestPlugin/extension.ts

```diff
@@ -54,7 +54,7 @@ async function analyzeSwagger(
   autoRestApi: extensionBase.Host
 ): Promise<void> {
   const swaggerFile = await autoRestApi.ReadFile(swaggerFileName);
-  const swagger = yaml.safeLoad(swaggerFile);
+  const swagger = yaml.load(swaggerFile);
```

Comment:
Is the old api deprecated?

Reply:
yes, it's supposed to use `load` in the new version of js-yaml